README.md

```diff
@@ -16,7 +16,8 @@ For the software developer, these common endpoints and patterns make it easier t
 |--------|-------------------------|--------------------------|-------------------|
 | **master**: the current release | [HTML](https://ga4gh-rnaseq.github.io/schema/master/) | [ReDoc](https://ga4gh-rnaseq.github.io/schema/master/redoc-ui.html) |<img src="http://validator.swagger.io/validator?url=https://raw.githubusercontent.com/ga4gh-rnaseq/schema/master/rnaget-openapi.yaml" /> |
 | **develop**: stable development branch | [HTML](https://ga4gh-rnaseq.github.io/schema/develop/) | [ReDoc](https://ga4gh-rnaseq.github.io/schema/develop/redoc-ui.html) |<img src="http://validator.swagger.io/validator?url=https://raw.githubusercontent.com/ga4gh-rnaseq/schema/develop/rnaget-openapi.yaml" /> |
-| **release/1.0.0**: RNAget version 1.0.0 | [HTML](https://ga4gh-rnaseq.github.io/schema/release/1.0.0/) | [ReDoc](https://ga4gh-rnaseq.github.io/schema/release/1.0.0/redoc-ui.html) |<img src="http://validator.swagger.io/validator?url=https://raw.githubusercontent.com/ga4gh-rnaseq/schema/release/1.0.0/rnaget-openapi.yaml" /> |
+| **release/1.1.0**: RNAget version 1.1.0 | | [ReDoc](https://ga4gh-rnaseq.github.io/schema/release/1.1.0/docs) |<img src="http://validator.swagger.io/validator?url=https://raw.githubusercontent.com/ga4gh-rnaseq/schema/RNAget-1.1.0/rnaget-openapi.yaml" /> |
+| **release/1.0.0**: RNAget version 1.0.0 | [HTML](https://ga4gh-rnaseq.github.io/schema/release/1.0.0/) | [ReDoc](https://ga4gh-rnaseq.github.io/schema/release/1.0.0/redoc-ui.html) |<img src="http://validator.swagger.io/validator?url=https://raw.githubusercontent.com/ga4gh-rnaseq/schema/RNAget-1.0.0/rnaget-openapi.yaml" /> |
 
```

Comment:
I haven't realized we were using a `release/1.0.0` branch for the validation URL. Using git tags is way better as we definitely we want to drop `release` branches.